generators/app/promptingHelpers.js

```diff
@@ -37,12 +37,12 @@ helper.sanitizePackageName = function (appName) {
     .toLowerCase()
 }
 
-helper.countryValidation = function (input) {
-  if (!/^[A-Z]{2}$/.test(input)) {
-    return 'Incorrect input please enter in this format (e.g. US, CA)'
-  }
-  return true
-}
+// helper.countryValidation = function (input) {
+//   if (!/^[A-Z]{2}$/.test(input)) {
+//     return 'Incorrect input please enter in this format (e.g. US, CA)'
+//   }
+//   return true
+// }
```

Comment:
why remove this?

if it's not useful / not used, we should just delete it, not comment it out